paddle/fluid/framework/details/inplace_op_pass.cc

```diff
@@ -301,6 +275,21 @@ std::unordered_set<ir::Node *> InplacePass::FindNodesByName(
   return ret;
 }
 
+void InplacePass::CollectInputAndOutputArgsOfOpDesc(
+    const OpDesc *op_desc, std::unordered_multiset<std::string> *in_args,
+    std::unordered_multiset<std::string> *out_args) {
+  in_args->clear();
+  out_args->clear();
+
+  for (auto &in_name : op_desc->InputArgumentNames()) {
+    in_args->insert(in_name);
+  }
+
+  for (auto &out_name : op_desc->OutputArgumentNames()) {
+    out_args->insert(out_name);
+  }
```

Comment:
Why the outputs of some op may have the same name?

Reply:
Just to check wrong case. I think you are right. We should add these check when converting program to graph.

Reply:
Done.